From db5bb393971a13ad9d0629561603b0c2ca077d4b Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Tue, 17 Sep 2024 21:47:36 +0200
Subject: [PATCH] boost std aligned alloc

---
 src/Makevars.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makevars.in b/src/Makevars.in
index 8c3a103..31ab9bf 100644
--- a/src/Makevars.in
+++ b/src/Makevars.in
@@ -1,4 +1,4 @@
-PKG_CPPFLAGS = -I./lib @cflags@
+PKG_CPPFLAGS = -I./lib -DBOOST_ASIO_DISABLE_STD_ALIGNED_ALLOC @cflags@
 PKG_LIBS += @libs@
 
 #### Debugging flags ####
